corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -934,7 +934,7 @@ hqDefine("cloudcare/js/form_entry/entries", [
         self.uploadFile = function () {
             document.getElementById(self.entryId).click();
         };
-        self.fileNameDisplay = ko.observable(constants.NO_FILE_SELECTED);
+        self.fileNameDisplay = ko.observable(gettext("No file selected."));
```

Comment:
I think you could have kept the contant

Reply:
I figured this particular string isn't going to be used anywhere else in the future so I'd just hard define them where they're needed and not an unnecessary line to the const.js file